### DISCUSSION TITLE
10 seconds 6FPS video to 60FPS 1 second video

### DISCUSSION THREAD

**Discussion by MangerzArt** [mifi/lossless-cut#1710] [Q&A]
ffmpeg -itsscale 0.1 -i 0.mp4 -c copy -an 0.mkv
Its it possible? or maybe i can add the comand somewhere

**Comment by mifi** [comment]
interesting! just tested on an mp4 file and it seems to work. (it even works on audio) I think we could implement a "speed up/slow down" function in losslesscut!

**Reply by mifi** [reply]
i've created a feature request: #1712
